app/javascript/components/widgets/weather/sky-icons.jsx

```diff
@@ -43,6 +78,13 @@ SkyIcon.propTypes = {
     'partly-cloudy-day',
     'partly-cloudy-night',
   ]).isRequired,
+  width: PropTypes.string,
+  height: PropTypes.string,
+};
+
+SkyIcon.defaultProps = {
+  width: '32px',
+  height: '32px',
```

Comment:
What do you think about:
```
export const SmallSkyIcon = (props) => <SkyIcon width={'32px'} height={'32px}' {...props} />;
export const LargeSkyIcon = (props) => <SkyIcon width={'72px'} height={'72px'} {...props} />;

export default SkyIcon
```
rather than specifying defaults?